h5pxblock/static/js/src/h5pxblock.js

```diff
@@ -1,85 +1,131 @@
 /* Javascript for H5PPlayerXBlock. */
 function H5PPlayerXBlock(runtime, element, args) {
-
+  function initH5P(H5PStandalone, service) {
     if (!window.H5PPlayerXBlockPromises) {
-        window.H5PPlayerXBlockPromises = [];
+      window.H5PPlayerXBlockPromises = [];
     }
-    const contentUserDataUrl = runtime.handlerUrl(element, 'user_interaction_data');
-    const contentxResultSaveUrl = runtime.handlerUrl(element, 'result_handler');
+    const contentUserDataUrl = runtime.handlerUrl(
+      element,
+      "user_interaction_data"
+    );
+    const contentxResultSaveUrl = runtime.handlerUrl(element, "result_handler");
+
+    const playerPromise = async function edXH5PPlayer(el) {
+      if (el && $(el).children(".h5p-iframe-wrapper").length == 0) {
```

Comment:
```suggestion
      if (el && $(el).children(".h5p-iframe-wrapper").length === 0) {
```
The .length == 0 check can be replaced with .length === 0 for a strict comparison, which is a recommended practice in JavaScript to avoid unexpected type conversions.

Reply:
Same comment as below